ImportExport/Strategy/AttributeImportStrategy.php

```diff
@@ -123,12 +123,6 @@ private function checkClassProperties(FieldConfigModel $entity): ?FieldConfigMod
             ) {
                 return null;
             }
-
```

Comment:
Makes sense to keep a BC layer here. We can check if `Akeneo_Aken_1706289854` field exists and apply this logic, and fallback to `Akeneo_bran_1706289854` otherwise